install.rst

```diff
@@ -92,6 +92,13 @@ on which version you’re using), and includes a complete Zeek environment with
 See our `Binary Packages wiki page <https://github.com/zeek/zeek/wiki/Binary-Packages>`_
 for the latest updates on binary releases.
 
+.. note:: In the past we also provided packages always referring to the latest
+   feature and LTS releases (``zeek`` and ``zeek-lts``, respectively). These
```

Comment:
This still holds for the feature release IIUC, otherwise how would you install Zeek 6.1 - that continues to be the `zeek` package and maybe should stay on line 75 above.

Reply:
Perhaps I'm confused — I thought when 6.1 comes out, we'll push `zeek-6.1` so you can say e.g. `apt install zeek-6.1` for the latest in that line of releases. I thought both `zeek` and `zeek-lts` won't be pushed any longer, since they both have the problem of switching release lines at a time of our choosing.

Reply:
Hmm, my statement was based on @0xxon 's comment [elsewhere](https://zeekorg.slack.com/archives/GUCJ5GH5M/p1688973606900199?thread_ts=1688712362.274209&cid=GUCJ5GH5M):
> Johanna
> Well, I am happy to make a zeek-5.0 then

> Johanna
> I don’t want to do packages for the feature releases though :slightly_smiling_face:

> Johanna
> that would get cluttered quick


So my understanding of is different.  Latest feature release (and also x.0 when overlapping) will continue to be available as `zeek` package only. The new `zeek-x.0` packages are only available for LTS releases.

Reply:
Yup, my understanding was the one of Arne. I am happy to push `zeek-x.0` packages for the LTS releases - but I want to avoid pushing our feature releases too, for several reasons.

So my current plan would be to have a `zeek` package for the current feature release (and people that just want to update automatically), and `zeek-x.0` packages for people who want to install the last LTS release.

Reply:
> So my current plan

I actually like that plan a lot. Any `x.1` or `x.2` feature release goes out of maintenance on the next release immediately, so the choice for users should either be:
* Run the latest feature release via the `zeek` package
* Run the latest LTS release with explicit version, the `zeek-x.0` package. Preferably the latest one, but prior ones will be around.
 
...and prior ones might even be supported for the short period where Zeek 5.0 and Zeek 6.0 (currently) are both receiving updates still. Currently we do not have a good packaging story for that, but with zeek-6.0 and zeek-7.0 existing there will be.

Reply:
I see, thanks. I like this as a forcing function toward the LTS releases, and as a signal that x.1/x.2 are short-lived.

But I also think our limited support windows for x.1/x.2 and pulling people across release lines are separate concerns, and ideally we'd address both needs ("stick to x.y, even if I fall out of maintenance", which favors stability, vs "just give me latest", which favors development). Then again it's reasonable for us to argue that folks wanting stability are better off with "pinned" LTS lines overall, plus installations that rarely update (to whatever version, including latest x.y) will generally need to look closely at what it means to upgrade when they do, anyway.

On a related note, if we were to start publishing migration guides, I'd be strongly in favor of doing those only for LTS releases, so your plan aligns with that.

So — I remain a bit torn, but I'm fine with it. I'll update the wording.